src/chatlog/chatlog.cpp

```diff
@@ -456,11 +456,18 @@ void ChatLog::startResizeWorker()
             workerAnchorLine = visibleLines.first();
     }
 
+    // switch to busy scene displaying the busy notification if there is a lot
+    // of text to be resized
+    int txt = 0;
+    for(ChatLine::Ptr line : lines)
```

Comment:
Why not short circuit these at 500,000 or whatever limit? If that chat is indeed very large, then just counting it could take a while.